packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -16,17 +16,18 @@
 
 import * as api from '@opentelemetry/api';
 import {
+  CompositePropagator,
   ConsoleLogger,
-  HttpTraceContext,
   HttpCorrelationContext,
-  CompositePropagator,
+  HttpTraceContext,
 } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
 import { SpanProcessor, Tracer } from '.';
-import { DEFAULT_CONFIG } from './config';
 import { MultiSpanProcessor } from './MultiSpanProcessor';
 import { NoopSpanProcessor } from './NoopSpanProcessor';
 import { SDKRegistrationConfig, TracerConfig } from './types';
-import { Resource } from '@opentelemetry/resources';
+import merge = require('lodash.merge');
+import { DEFAULT_CONFIG } from './config';
```

Comment:
organized imports.

N.B. `import merge = require('lodash.merge');` because to `import from` would require `esModuleInterop ` to be `true` due to the nature of lodash single function exports.